### PR TITLE
AC-2487::Accordions do not indicate current state

### PR DESCRIPTION
**Pull request by glo82145**
## Description

Accordions do not indicate current state.

Environment
Adobe Magento - Venia

Context
Windows 10; Chrome 88; JAWS 2021

Reproduction Steps
Locations (representative sample):

Search Results
Account - Order History
1. With JAWS operating, press Tab to move through the content.
2. When keyboard focus is on "Filter" button, press Enter or Space to activate it.
3. Within "Filter" menu tab to filters, press Enter or Space to activate accordion.

Actual Behavior
Accordions do not indicate current state information (expanded or collapsed). Examples include:

Module 03 - Search Results
Module 08 - Account - Order History

When controls do not provide state information, screen reader users will not know their current state.

Expected Behavior
For each control (accordions) that have multiple states (ie. expanded, collapsed), ensure the state is communicated to let the user know what will occur when they interact with the control. Use the aria-expanded attribute, set to true or false, to identify the current state.

## Related Issue

Closes [#AC-2487.](https://jira.corp.adobe.com/browse/AC-2487)

## Checklist

-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.

### Resolved issues:
1. [x] resolves magento/pwa-studio#3990: AC-2487::Accordions do not indicate current state

### CODE REVIEW

**Comment by anthoula** [comment]
@magento create issue from PR

File: packages/venia-ui/lib/components/FilterModalOpenButton/filterModalOpenButton.js [hunk 1]
```diff
@@ -19,9 +24,11 @@ const FilterModalOpenButton = props => {
             }}
             data-cy="FilterModalOpenButton-button"
             onClick={handleOpen}
+            onKeyDown={handleKeypress}
             type="button"
             aria-live="polite"
             aria-busy="false"
+            aria-label="Filter Button for Filter Options"
```
Comment (anthoula) [inline packages/venia-ui/lib/components/FilterModalOpenButton/filterModalOpenButton.js:31]: Can you add a translation for this string please?

**Review by anthoula — CHANGES_REQUESTED** [review]
Thanks @glo82145 - Just some translations needed please

**Review by anthoula — DISMISSED** [review]

**Comment by dpatil-magento** [comment]
run cypress

**Comment by dpatil-magento** [comment]
lighthouse-desktop

**Comment by dpatil-magento** [comment]
run lighthouse-mobile

**Comment by dpatil-magento** [comment]
QA Approved.

**Pull request merged**